pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -109,6 +109,10 @@ func NewEndpointController(ctx context.Context, podInformer coreinformers.PodInf
 	e.podLister = podInformer.Lister()
 	e.podsSynced = podInformer.Informer().HasSynced
 
+	// Initialize the pod indexer
+	controller.AddPodNamespaceLabelIndexer(podInformer.Informer()) //nolint:errcheck
```

Comment:
Why not checking error?

Reply:
We do the same in multiple controller today, following the same.